src/ui/Dropdown.css

```diff
@@ -76,13 +76,19 @@
    default focus styles. This is better for accessibility. */
 @media screen and (-webkit-min-device-pixel-ratio:0) {
   .root select {
-    opacity: 1;
     color: transparent;
+    opacity: 1;
```

Comment:
Why did we move this line? Maybe it's not intentional? In the effort of keeping changes clean and minimal, best to only change the lines that need changing. This helps preserve diff blame.